src/lib/preferences/preferences.cpp

```diff
@@ -214,7 +214,13 @@ Preferences::Preferences(BrowserWindow* window)
     //APPEREANCE
     settings.beginGroup("Browser-View-Settings");
     ui->showStatusbar->setChecked(settings.value("showStatusBar", false).toBool());
+    // NOTE: instantBookmarksToolbar and showBookmarksToolbar cannot be both enabled at the same time
```

Comment:
@innermous 
Because it's a mockup of screencaps and not Code ... harder to redraw. ;) That could work too. That would remove the CON of loss of screen real estate for sure.